Dockerfile

```diff
@@ -4,12 +4,21 @@ LABEL MAINTAINER Quartile Limited <info@quartile.co>
 ARG ODOO_SOURCE=OCA/OCB
 ARG ODOO_VERSION=10.0
 
-RUN apt-get update && apt-get install -y gnupg
+# Update and install all necessary packages
+RUN apt-get update && apt-get install -y \
+    wget \
+    ca-certificates \
```

Comment:
@AungKoKoLin1997 What is this for?

Reply:
We need to use this for archived repo because this can't be accessed directly and get access denfined. If not, we will get below error.

```
3.096 W: https://apt-archive.postgresql.org/pub/repos/apt/dists/bionic-pgdg/InRelease: No system certificates available. Try installing ca-certificates.
3.096 W: https://apt-archive.postgresql.org/pub/repos/apt/dists/bionic-pgdg/Release: No system certificates available. Try installing ca-certificates.
3.096 E: The repository 'https://apt-archive.postgresql.org/pub/repos/apt bionic-pgdg Release' does not have a Release file.
```

Reply:
@AungKoKoLin1997 Can you add a comment in the code, then, please.